R/pkg/R/stats.R

```diff
@@ -52,22 +52,17 @@ setMethod("crosstab",
             collect(dataFrame(sct))
           })
 
-#' Calculate the sample covariance of two numerical columns of a SparkDataFrame.
```

Comment:
The method for SparkDataFrame is still there. I'm just removing redundant doc here.